docs/content/contributing/issues-and-pull-requests.md

```diff
@@ -35,8 +35,7 @@ The development workflow is the following:
    2. Run `git submodule update --init --recursive`
 2. Write the code that is needed :D
 3. Add integration tests if necessary
-4. Get the linters with `make install_linters` and install `jq` with the package manager of your OS
-5. Use `make clean all` to build image locally and run tests (note that tests work on Linux **only**)
```

Comment:
I assume the `clean` task was not required to be added prior to `all` as it runs at the end of `all`? Was there any other reason this has been dropped from the instructions?

I'm assuming that `clean` was advised this way in the event that something went wrong and broke during `make`, and that prevented `clean` from running at the end? If that's possible, it'd make sense to want to have a clean state before running `make` again? Should this advice really be dropped?

Additionally while this PR adds testing support for macOS, does that warrant dropping our compatibility notice? Should it remain explicit that Windows is not supported (or that only macOS and Linux work for testing?), assuming that is the case, eg WSL isn't supported or any other approach a contributor on Windows might try?

Reply:
I've run this multiple times successfully on WSL - so the question is if we should keep the focus on Linux (only!) and just give some hints on what to do on other platforms, what might work?

Reply:
If there are no known issues, it's fine then. Can add back the disclaimer if someone finds a problem and we're unable to support that for some reason.

Reply:
@wernerfred Correct me, if I am wrong. WSL2 is basically a Ubuntu VM. So the tests still run on linux, even if the host is Windows.

Reply:
Just wanted to respond to:
> eg WSL isn't supported or any other approach a contributor on Windows might try
imho we should stay with linux only BUT add hints on what do to on other plattforms. Here would be the hint to use WSL on windows as in fact then it runs on linux.

Reply:
Good catch @polarathene , I've updated the docs to include clean again.